gradle/versions.gradle

```diff
@@ -8,7 +8,7 @@ ext {
     buildToolsVersion = "28.0.3"
```

Comment:
I see the build tools were updated to `29.0.3` on `travis.yml`. How comes we are not doing the same here?

Reply:
The build tools version is ignored, in any event.

Reply:
Ah. I see. Thank you for the clarification.

Reply:
Eh - we should really clean up the Gradle files.  If this works, I may clean out some of these warnings before it merges.